### ISSUE TITLE
modernise this project, get working again with current imager

### ISSUE THREAD

**Issue by gijzelaerr** [transientskp/aartfaac-website#1]

**Comment by Error323** [comment]
The current imager is now completely independent of the website. The website should only host the youtube live feed, this will simplify things.

**Comment by gijzelaerr** [comment]
right, so it just a bunch of static pages then. I'll rewrite it to a github hosted website, with all static pages. agree?

**Comment by Error323** [comment]
agreed!

**Comment by gijzelaerr** [comment]
rewrite of the website in jekyll can be found in the v2 branch:

https://github.com/aartfaac/aartfaac.github.com/tree/v2

jekyll is quite simple to use and is the default template rendering agent on github:

https://jekyllrb.com/docs/quickstart/
https://pages.github.com
